e2e/cypress/e2e/runner/initialiseSession.js

```diff
@@ -29,5 +29,5 @@ When("I revisit the status page", () => {
 
 When("I declare and continue", () => {
   cy.findByLabelText("Confirm").click();
-  cy.findByRole("button", { name: /submit/i }).click();
+  cy.findByRole("button", { name: /Save and continue/i, exact: false }).click();
 });
```

Comment:
A little confused by the API of `findByRole` - to me it seems it has 3 different ways of checking case specific matching?

Either regex `/Save and continue/i`, `exact: false` or `matchCase: false` - I don't see `exact` or `matchCase` here either... https://github.com/testing-library/dom-testing-library/blob/a7b725257e61370675ee043813003d36fae23e9d/types/queries.d.ts#L69-L138 But not sure if the types are from elsewhere or something 🤔 

Not specific to your PR, just something that popped in my head while reading this.